Moodis/Moodis/Feature/Group/GroupViewModel.cs

```diff
@@ -0,0 +1,245 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Moodis.Feature.Login;
+
+namespace Moodis.Feature.Group
+{
+    class GroupViewModel
+    {
+        public static Group group;
+
+        public Group createGroup(string name)
+        {
+            group = new Group(name, SignInViewModel.currentUser.username);
+
+            var groupList = new List<Group>();
+            var userList = new List<User>();
+
+            if (File.Exists(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData) + "/groups.bin"))
+            {
+                groupList = Serializer.Load<List<Group>>(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData) + "/groups.bin");
+            }
+
+            if (!groupList.Exists(x => x.name == name && x.manager == SignInViewModel.currentUser.username))
+            {
+                groupList.Add(group);
+                Serializer.Save(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData) + "/groups.bin", groupList);
+            }
+
+            if (File.Exists(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData) + "/users.bin"))
+            {
+                userList = Serializer.Load<List<User>>(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData) + "/users.bin");
+            }
+
+            if (userList.Exists(user => user.username == SignInViewModel.currentUser.username))
+            {
+                int index = userList.FindIndex(user => user.username == SignInViewModel.currentUser.username);
+                userList[index].groupName = name;
+                SignInViewModel.currentUser.groupName = name;
+                Serializer.Save(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData) + "/users.bin",userList);
+            }
+            return group;
```

Comment:
won't this always be true ? If i'm correct it checks whether the current user exists, which is always true.